docs/debian-conf.d/main/00_vexim_listmacrosdefs

```diff
@@ -42,3 +42,9 @@ VEXIM_LOCALPART_SUFFIX = +*
 
 CHECK_RCPT_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-rcpt.conf
 CHECK_DATA_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-content.conf
+CHECK_HELO_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-helo.conf
```

Comment:
I just placed `vexim-acl-check-helo.conf` where the other `vexim-acl-check-*.conf` have been placed. For debian it would make more sense to place them into the `acl`-folder. But then we would have duplicates (or can we use links in github?).

I used a separate file because we did the same in the other configuration file.